### PR TITLE
Updated translations.

### PR DESCRIPTION
**Pull request by NeonFrights**
I'm focusing on updating and expanding on the translations provided by the machine translator. I'll make it a personal project to cover as much as possible during my free time over the next few months.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/list/h/sound/voice/30/hvoice_c07_02_01/translation.txt [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+焦らしてんだよ。すぐにイカせたら面白くねぇだろ？=I'm just playing with you. It wouldn't be as much fun if I made you nut right away, would it? No, I'm gonna take my time.
+んっ、あくっ……どうした？　キツ過ぎて動けねぇか？=What's the matter? Is it too tight for you to move around?
+んんぅっ、こんなもんか？　はぁっ、もっとできるだろ？=Uhhh... is that it? C'mon, you can do better. I know you can.
+んっ、じっくりいたぶってやるよ……その方が嬉しいだろ？=I'll take my time with you. I'll get you begging me to let you cum... then maybe I'll let you.
+はぁっ、オレに遊ばれる気分はどうだ？　悪くないだろう？=Ahhh, having your dick worked on by me. You're a very lucky boy, aren't you?
+ほら、オレの尻はどうだ？　んっ、ゆっくり味わってけよ……=Are you enjoying me there? Go easy and I'll make it better for you.
+やっぱ尻はキツいか？　ゆっくりでも、かなり効くだろう？=Take it a little easier with my butt, remember it's a tighter fit!
+んっ、ちまちま擦ってるだけで……こんなのどこがいいんだ？=Just rubbing a little, but why does it feel so good?
+はぁっ、微妙に感じるが……オレはもっと強い方が好みだな。=It's good, but I want you to go harder!
+んんっ、動けない分……オマエの動きがもろに分かるな……（オマエのがどんどん口の中で大きく）=
```
Comment (SpockBauru) [inline Translation/en/RedirectedResources/assets/abdata/list/h/sound/voice/30/hvoice_c07_02_01/translation.txt:16]: More uncommented and untranslated lines...

**Review by SpockBauru — CHANGES_REQUESTED** [review]
There are some major issues regarding lines that are both uncommented and untranslated. I cannot merge without fixing these issues.
Overall, I like you interpretation of the Sadistic character. She’s dirty, sarcastic, and somewhat aggressive, a true dominatrix. But sometimes you are getting “too far”, I recommend you make a revision and tone down the parts that are too much out of context (there are not many of them).
There are several punctuation and uppercase misses, please consider use the Style Check tool: https://github.com/SpockBauru/TranslationTools_Illusion

**Comment by NeonFrights** [comment]
> There are some major issues regarding lines that are both uncommented and untranslated. I cannot merge without fixing these issues. Overall, I like you interpretation of the Sadistic character. She’s dirty, sarcastic, and somewhat aggressive, a true dominatrix. But sometimes you are getting “too far”, I recommend you make a revision and tone down the parts that are too much out of context (there are not many of them). There are several punctuation and uppercase misses, please consider use the Style Check tool: https://github.com/SpockBauru/TranslationTools_Illusion
> 
> Tip: GitHub is hiding many conversations, click in "Load more" when you find bellow.

Thank you for your response! I will look over the suggested changes, especially regarding the use of trademarked material, as I continue the translation efforts. I am also making an effort to add as much definable personality as possible rather than have the girl simply serve as an object, for this reason I am adding conversational cues that don't necessarily relate to the core objective of getting with her. I have also had issues with context as the (machine) translations do not denote the specific scenes involved and I only recognise the translations are misplaced when they display in-game in the unintended scene(s). As for punctuation, I have missed some parentheses to indicate she is communicating through her thoughts, but uppercase following an ellipsis in standard English is only appropriate when it indicates the beginning of a new sentence - if what follows the ellipsis is a continuation of the same sentence, it must be lowercase! Thank you again for your feedback, I look forward to you reviewing the completed project... however, I don't know yet if I will attempt any of the other personalities.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/30/00/2/translation.txt [hunk 2]
```diff
@@ -2,17 +2,17 @@
...
 //attitude:neutral
-//ほう、オレを誘うとは、大した度胸だな。=
-//いいだろう。だが、誘ったからにはしっかりやれよ？=
+ほう、オレを誘うとは、大した度胸だな。=You got balls to ask me that. I'm intrigued to see what you can do.
+いいだろう。だが、誘ったからにはしっかりやれよ？=That direct, huh? I'll give you a shot, but do you know what you're doing?
 //attitude:favor
-//ああ、別にいいぞ。断る理由も無いしな。=
-//顔を真っ赤にしてお誘いか？　断ったら泣いちまいそうだな。=
+ああ、別にいいぞ。断る理由も無いしな。=I'm making a point of saying yes to suggestions like that more often.
+顔を真っ赤にしてお誘いか？　断ったら泣いちまいそうだな。=Sure! I'll let you put it in me if you want, but you have to let me sit on your face in return.
```
[the remark this answers is not in the document]
Comment (NeonFrights) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/30/00/2/translation.txt:9]: Thank you! I don't know any Japanese and Google Translate doesn't help much, the original text seems very informal and full of slang that Translate doesn't recognise in context, which would be expected.

**Comment by NeonFrights** [comment]
Hi again! Just to enquire, do you have a list of the scenarios occurring for each number within /c07/30, /34, and /50, along with the h scenes for /hvoice_c07... ? It would certainly make things a lot easier!

**Comment by SpockBauru** [comment]
> Hi again! Just to enquire, do you have a list of the scenarios occurring for each number within /c07/30, /34, and /50, along with the h scenes for /hvoice_c07... ? It would certainly make things a lot easier!

I wish I have something like this... If you manage to make it, please share! It will help a lot!

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/30/31/10/translation.txt [hunk 3]
```diff
@@ -1,8 +1,8 @@
 //
 // Dumped for Honey Select 2 v1.1.3 by Text Dump v1.4.1
 //
-//まさかトイレでヤることになるとはな。オレまで変態の仲間入りだ……=
-//まぁ、出すもん出さなきゃいけないのは誰だって同じだ。濃いか薄いかの違いしかないなら、トイレでしたって変じゃあねぇよな。=
-//なんでこう、トイレってのはムラムラするんだろうな。出すもん出すところが、同じってだけなのによ。=
-//オマエ、便所の才能があるんじゃねぇか？　良かったな、一つでも自慢できることが見つかって。=
-//他人の排泄に興奮するってのは、そう珍しいことでもねぇだろ。よくある変態趣味の一つだ。=
+まさかトイレでヤることになるとはな。オレまで変態の仲間入りだ……=Fucking him in the toilet, and liking it... so this is how I learn how much of a pervert I am...
```
[the remark this answers is not in the document]
Comment (NeonFrights) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c07/30/31/10/translation.txt:4]: It certainly should! Will have those little details fixed for the next commit submission!

**Pull request merged**